### DISCUSSION TITLE
Domain redirection issue

### DISCUSSION THREAD

**Discussion by gprao08** [Sunbird-cQube/community#863] [General]
@RNarmatha 
When we hit **_https://vsk.ap.gov.in_**    it will redirect to Minio Bucket console instead of login screen. AP School Education Department asking that without providing login credentials it should not go to the dashboard screen directly (if we hit https://vsk.ap.gov.in/dashboard).  Could you plz reply?

**Comment by bommalla** [comment]
@gprao08 

Thanks for reaching us

**Comment by bommalla** [comment]
@gprao08 

In cQube version 5.0.3, accessing the URL directly in the browser would display the login page, but in the latest version, cQube v5.0.5, it redirects to a Minio bucket as per the development process. It's preferable to use the URL format https://vsk.ap.gov.in/dashboard when accessing the latest version of cQube.

**Comment by bommalla** [comment]
@gprao08 

In the latest version of cQube (v5.0.5), there are two types of users: private users and public users. If you need access to the login dashboard, you can directly log in using the private user credentials. Please find below the URLs highlighting the differences between public and private users.

https://vsk.ap.gov.in/dashboard/#/summary-statistics -------- This is public

https://vsk.ap.gov.in/dashboard/#/login  ------ This is private
